src/app/content/hooks/locationChange/resolveContent.ts

```diff
@@ -150,27 +150,39 @@ const resolvePage = async(
 
 const getBookInformation = async(
   services: AppServices & MiddlewareAPI,
-  pageId: string
+  reference: {
+    bookId?: string,
+    bookVersion?: string,
+    pageUid: string,
```

Comment:
can you go back and rename `pageUid` to `pageId` so its consistent with the standard naming? i know it was like that already but while we're in here lets fix it